RallyHereIntegration/Source/RallyHereIntegration/Private/RH_SessionHelpers.h

```diff
@@ -852,3 +869,29 @@ class FRH_SessionJoinByPlatformIdHelper : public FRH_SessionLookupHelper
 
 	FRH_OnSessionUpdatedDelegateBlock Delegate;
 };
+
+
+// Wrapper around a delete that will remove the session locally once the delete is completed, without requiring a poll on the session
+class FRH_SessionLeaveHelper : public FRH_SessionRequestAndModifyHelper<RallyHereAPI::Traits_LeaveSessionByIdSelf>
+{
+public:
+	typedef RallyHereAPI::Traits_LeaveSessionByIdSelf BaseType;
+
+	FRH_SessionLeaveHelper(FRH_SessionOwnerPtr InSessionOwner, const FString& InSessionId, FRH_OnSessionUpdatedDelegateBlock InDelegate = FRH_OnSessionUpdatedDelegateBlock(), int32 InPriority = DefaultRallyHereAPIPriority)
+		: FRH_SessionRequestAndModifyHelper(InSessionOwner, InSessionId, InDelegate, InPriority)
+	{
+	}
+
+	virtual void OnSessionPollComplete(bool bSuccess, bool bResetTimer)
```

Comment:
I highly suggest using `override` instead of `virtual` when you know you are expecting to override a function instead of declaring a new virtual function

Reply:
I usually do, missed it here (though I usually have both, even if virtual is not required)